src/services/leafletMapDefaults.js

```diff
@@ -101,69 +102,8 @@ angular.module('ui-leaflet').factory('leafletMapDefaults', function ($q, leaflet
         setDefaults: function (userDefaults, scopeId) {
             var newDefaults = _getDefaults();
 
-            if (isDefined(userDefaults)) {
```

Comment:
Yes merge is for complex, but it will limit us to angular 1.4, which I am fine w/